lecture10 template file/lecture10-alb.inport.yaml

```
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: lecture10 alb
+
+Resources:
+
+ alb:
+   Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+   Properties: 
+     IpAddressType: ipv4
+     Name: lecture10
+     Scheme: internet-facing
+     SecurityGroups: 
+       - {'Fn::ImportValue': 'PublicSecurityGroup'}
```

これはJSON記法なのでYAMLにする
```suggestion
       - ???
```
組み込み関数の方針として省略記法で書くのかどうかも定めておくこと（Fn、!）